lab_6_pipeline/pos_frequency_pipeline.py

```diff
@@ -2,6 +2,11 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 from typing import Optional
+from lab_6_pipeline.pipeline import *
+
+
+class EmptyFileError:
+    """sfdf"""
```

Comment:
Is it required somewhere?

Reply:
It is required to define ```EmptyFileError``` in pos_frequency_pipeline file to execute checks locally